src/otaclient/ota_core.py

```diff
@@ -98,12 +99,14 @@
 DEFAULT_STATUS_QUERY_INTERVAL = 1
 WAIT_BEFORE_REBOOT = 6
 DOWNLOAD_STATS_REPORT_BATCH = 300
-DOWNLOAD_REPORT_INTERVAL = 1  # second
+DOWNLOAD_REPORT_INTERVAL = 3  # second
```

Comment:
FYI. currently, Edge check status every second, then report it to backaend every 10 seconds.